LocalSettings.php

```diff
@@ -2225,6 +2225,10 @@
 			'batfamilywiki',
 			'batmanwikifandom',
 			'd',
+			
+		],
+		'+hypixelwiki' => [
+			'hypixelwikifandom',
```

Comment:
Interwiki prefix does not exist. Would you like me to create it on your wiki?

Reply:
Yes, please.

Reply:
What is the URL to your Fandom wiki?

Reply:
minecraft.fandom.com

Reply:
So not https://hypixel.fandom.com ?

Reply:
wait I mean hypixel

Reply:
Sorry, I was looking at the minecraft fandom recently to check out CSS.

Reply:
Done.

Reply:
@Universal-Omega Technically this didn't need an interwiki prefix added, as `wikia:hypixel` exists globally. :)

Reply:
Oh. My bad then.

Reply:
> Oh. My bad then.

@Universal-Omega No problem. Both work. It would just save a step. I just thought I'd point that out for the future.